docs/writing-tests/interaction-testing.md

```diff
@@ -6,17 +6,52 @@ As you build more complex UIs like pages, components become responsible for more
 
 In a nutshell, you start by supplying the appropriate props for the initial state of a component. Then simulate user behavior such as clicks and form entries. Finally, check whether the UI and component state update correctly.
 
+In Storybook, this familiar workflow happens in your browser. That makes it easier to debug failures because you're running tests in the same environment as you develop components—the browser.
+
 ![Storybook interaction testing](./storybook-interaction-tests.gif)
 
-## Setup interactions addon
+## How does component testing in Storybook work?
 
-You can set up interaction testing in Storybook using the `play` function and [`@storybook/addon-interactions`](https://storybook.js.org/addons/@storybook/addon-interactions/).
+You start by writing a [**story**](../writing-stories/introduction.md) to set up the component's initial state. Then simulate user behavior using the **play** function. Finally, use the Storybook **test-runner** to verify the DOM structure. Additionally, you can automate test execution via the [command line](./test-runner.md#cli-options) or in your [CI environment](./test-runner.md#set-up-ci-to-run-tests).
 
 - The [`play`](../writing-stories/play-function.md) function is a small snippet of code that runs after a story finishes rendering. You can use this to test user workflows.
+- The test is written using Storybook-instrumented versions of [Jest](https://jestjs.io/) and [Testing Library](https://testing-library.com/).
+- [`@storybook/addon-interactions`](https://storybook.js.org/addons/@storybook/addon-interactions/) visualizes the test in Storybook and playback interface for convenient browser-based debugging.
+- [`@storybook/test-runner`](https://github.com/storybookjs/test-runner) is a standalone utility—powered by [Playwright](https://playwright.dev/)—that executes all your interactions tests and catches broken stories.
+
+## Set up the interactions addon
+
+To enable interaction testing with Storybook, you'll need to take additional steps to set it up properly. We recommend you go through the [test runner documentation](./test-runner.md) before proceeding with the rest of the required configuration.
```

Comment:
> We recommend you go through the [test runner documentation](./test-runner.md) before proceeding with the rest of the required configuration.

Why? I don't see how that's a prerequisite until the 'Execute tests with the test-runner' section, at which point this disclaimer would be redundant.

Reply:
@kylegach my reading on this is the following, this is a "mise en place" type of situation, and doesn't hurt to leave it here. As it indirectly pulls traffic into the test runner docs and also prevents unwanted issues here or questions in other platforms mentioning that they ran through the documentation and weren't able to get accessibility tests enabled by following the docs. I'm more than glad to follow up with you on this.